aspnet/signalr/overview/performance/scaleout-with-redis.md

```diff
@@ -74,11 +74,11 @@ Download the Ubuntu ISO from [http://www.ubuntu.com](http://www.ubuntu.com/).
 
 In Hyper-V, add a new VM. In the **Connect Virtual Hard Disk** step, select **Create a virtual hard disk**.
 
-![Screenshot of a window titled New Virtual Machine Wizard and Connect Virtual Hard Disk and Create a virtual hard disk are selected.](scaleout-with-redis/_static/image2.png)
+![Screenshot that shows the New Virtual Machine Wizard dialog box. Connect Virtual Hard Disk and Create a virtual hard disk are selected.](scaleout-with-redis/_static/image2.png)
 
 In the **Installation Options** step, select **Image file (.iso)**, click **Browse**, and browse to the Ubuntu installation ISO.
 
-![Screenshot that shows a window titled New Virtual Machine Wizard Installation Options is highlighted Install from a boot C D D V D ROM and Image file dot i s o are selected](scaleout-with-redis/_static/image3.png)
+![Screenshot that shows the New Virtual Machine Wizard on the Installation Options page. Install an O S from a boot C D D V D ROM and Image file dot i s o are selected](scaleout-with-redis/_static/image3.png)
```

Comment:
I didn't catch this the first time, but let's add a period to the end of your sentence.